src/lib/CollectStore.js

```diff
@@ -106,32 +107,45 @@ export default class CollectStore {
       .sort(sortByName)
   }
 
+  // Get the drive application url using the list of application
+  // FIXME this works only because we need the application list permission for the cozy-bar
+  // and this also supposes that the drive application has the 'drive' slug
+  fetchDriveUrl () {
+    return cozy.client.fetchJSON('GET', '/apps/')
```

Comment:
Actually, it depends on the use of this. As I use it in this PR, it is only called once and then there is no need to cache the result. It depends on the result of the other discussions in this PR on where this call should be done.